examples/long-short.js

```diff
@@ -379,13 +379,15 @@ class LongShort {
             const close = resp.results[0].c;
             resolve(close);
           } else{
-            const resp = await this.alpaca.getBars('minute',
-                stock,
-                { limit: 1 })
-            resolve(resp[stock][0].closePrice);
+            const resp = await this.alpaca.getBarsV2(stock, {
+              timeframe: '1Min',
+              limit: 1,
+            })
+            const bars = await generatorToArray(resp);
+            resolve(bars[0].ClosePrice);
           }
         } catch (err) {
-          log(err.message)
+          log(`Encountered error ${err.message} for ${JSON.stringify(stock)}.`)
```

Comment:
In debugging it helped a lot to know exactly for what stock the error happened, so I kept it in because I feel that it's an improvement on the previous error messaging